man/halt.xml

```diff
@@ -137,12 +138,34 @@
   <refsect1>
     <title>Notes</title>
 
-    <para>These commands are implemented in a way that preserves compatibility with
+    <para>These commands are implemented in a way that preserves basic compatibility with
     the original SysV commands.
     <citerefentry><refentrytitle>systemctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>
     verbs <command>halt</command>, <command>poweroff</command>,
     <command>reboot</command> provide the same functionality with some additional
     features.</para>
+
+    <para>One major difference in the behavior of these commands under
+    <literal>systemd</literal> and SysV systems
+    is that under SysV, the "<command>halt</command>" command typically caused the
```

Comment:
No need to do double differentiation here. Please drop the quotes from around `<command>`.